mmcls/evaluation/metrics/single_label.py

```diff
@@ -151,8 +167,10 @@ def process(self, data_batch, data_samples: Sequence[dict]):
             result = dict()
             pred_label = data_sample['pred_label']
             gt_label = data_sample['gt_label']
+            result['img_path'] = data_sample['img_path']
```

Comment:
If the `img_path` is used in the `mCE` metric, please add it to the result in `mCE`, because `Accuracy` doesn't need it.